src/lib.rs

```diff
@@ -258,65 +279,100 @@ impl Watch {
             };
         }
     }
+
+    fn is_excluded_path(&self, path: &Path) -> bool {
+        self.exclude_paths.iter().any(|x| path.starts_with(x))
+            || self
+                .workspace_exclude_paths
+                .iter()
+                .any(|x| path.starts_with(x))
+    }
+
+    fn is_hidden_path(&self, path: &Path) -> bool {
+        path.file_name()
+            .and_then(|x| x.to_str())
+            .map(|x| x.starts_with('.'))
+            .unwrap_or(false)
+    }
 }
 
+#[non_exhaustive]
 #[derive(Debug, StructOpt)]
 pub struct DevServer {
     #[structopt(long, default_value = "127.0.0.1")]
-    ip: IpAddr,
+    pub ip: IpAddr,
     #[structopt(long, default_value = "8000")]
-    port: u16,
+    pub port: u16,
 
     #[structopt(flatten)]
-    watch: Watch,
+    pub watch: Watch,
+    #[structopt(skip)]
+    served_path: Option<PathBuf>,
+    #[structopt(skip)]
+    release: bool,
```

Comment:
But then for consistency sake I would simply make all the fields public. The setter methods are there for convenience (they are nice especially for paths because they use `AsRef<Path>`)

Reply:
Solved by #14